src/components/buttons/primary-button/primary-button.visualspec.js

```diff
@@ -0,0 +1,13 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { widths } from '../../../../test/percy/widths';
```

Comment:
Why do we mostly import both withs and percySnapshots and not export a pre-filled/wrapped percySnapshot from the percy/utils? Would that shorten and centralise things a bit more?

Reply:
We had this in our setup before, where we would preconfigure the snapshot-fn with the widths. I think it makes sense to do the same thing here as well. Nice suggestions! 👍

Reply:
I think instead we can just use percy.yml and specify widths
[ref](https://docs.percy.io/v1/docs/configuration)

Reply:
I went for that approach instead so that it's easier to understand what's happening with the percySnapshot [ref](https://github.com/commercetools/ui-kit/pull/373/commits/8778d86143c47d93a0ce31a5751943e3543f6f7b)